prism/src/main/java/org/apache/falcon/resource/proxy/ExtensionManagerProxy.java

```diff
@@ -460,15 +473,9 @@ protected APIResult doExecute(String colo) throws FalconException {
                 }else{
                     processNames.add(entity.getName());
                 }
+                metaStore.storeExtensionJob(jobName, extensionName, feedNames, processNames, configBytes);
```

Comment:
In the first place, we have just entered empty list of process and feeds.  This is the step where we update that information after every successful entity submission.